PLAYGROUND.md

```diff
@@ -71,3 +71,22 @@ Open `glue.json`, and you'll see JSON similar to this file:
 }
 ```
 Each line represent one finding, and it's state. The state can be either `new` (Glue will report it), `ignore` (Glue will not report it) or `postpone:%d-%m-%Y` (Glue will ignore this issue until the specific date). This gives you the ability to ignore or postpone the issues found by Glue.
+
+####Docker example
+Another example might exclude specific findings:
+```
+	docker run --rm --name=Glue \
+		-v $(PWD):/tmp/triage owasp/glue \
+		-t sfl \
+		-t retirejs \
+		-t nodesecurityproject \
+		--finding-file-path /tmp/triage/glue_ignore.json  \
+		/tmp/triage
+```
+If there were several items reported which you believed were false-positives, `glue_ignore.json` might include the following:
```

Comment:
Maybe add a note explaning how to get those ids that you put in the `glue_ignore.json`?

Reply:
Done!